docker_test.go

```diff
@@ -1187,43 +1196,29 @@ func ExampleContainer_MappedPort() {
 	// }
 }
 
-func TestContainerCreationWithBindAndVolume(t *testing.T) {
+func TestContainerCreationWithVolumeAndFileWritingToIt(t *testing.T) {
 	absPath, err := filepath.Abs(filepath.Join(".", "testdata", "hello.sh"))
 	if err != nil {
 		t.Fatal(err)
 	}
 	ctx, cnl := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cnl()
-	// Create a Docker client.
```

Comment:
No need to instantiate a client for the volume, as Docker will handle the mounts correctly.